code/game/objects/structures/tables_racks.dm

```diff
@@ -850,14 +850,26 @@
 	if(O.loc != src.loc)
 		step(O, get_dir(O, src))
 
+/obj/structure/rack/proc/item_placed(item)
```

Comment:
Do we need this? It's also present on tables but only to be overriden by trays to track tray contents.